Description: Relocate non-libs from /usr/lib/X11
 For multiarch support, we change libdir to /usr/lib/$(DEB_HOST_MULTIARCH).
 However, we do not want to do this to /usr/lib/X11 yet.
 .
 Note that XMBINDDIR_FALLBACK should really be set to /usr/share/X11/bindings
 since the files are platform-independent. We postpone moving them until a
 decision has been reached on libmotif-common.
 .
 The configure.ac fixes are minimal and only suitable for Debian/Ubuntu
 packaging. For upstream one might want to add a command line option.
 .
 Sergio Gelato, 2012-05-20.
 .
 Allow the MWM resources and fallback bindings directories to be customized
 with the --with-mwmrcdir and --xmbinddir configure options respectively.
 Now we can relocate non-library files from /usr/lib/X11 to /usr/share/X11
 (bindings) and /etc/X11/mwm (MWM resources) in debian/rules, allowing us to
 split motif into architecture-dependent and architecture-independent packages.
 .
 Graham Inggs, 2013-03-12.
Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=673690
Forwarded: http://bugs.motifzone.net/show_bug.cgi?id=1581
Author: Sergio Gelato <Sergio.Gelato@astro.su.se>
Author: Graham Inggs <graham@nerve.org.za>
Last-Update: 2013-03-08
--- a/configure.ac
+++ b/configure.ac
@@ -178,13 +178,15 @@
 LIBDIR="${libdir}/X11"
 AC_SUBST(LIBDIR)
 
-MWMRCDIR="${libdir}/X11"
+AC_ARG_WITH(mwmrcdir, [  --with-mwmrcdir=DIR  mwm resources are located in DIR], mwmrcdir=$withval, mwmrcdir="${libdir}/X11")
+MWMRCDIR="$mwmrcdir"
 AC_SUBST(MWMRCDIR)
 
 INCDIR="${includedir}/X11"
 AC_SUBST(INCDIR)
 
-XMBINDDIR_FALLBACK="${libdir}/X11/bindings"
+AC_ARG_WITH(xmbinddir, [  --with-xmbinddir=DIR  fallback bindings are located in DIR], xmbinddir=$withval, xmbinddir="${libdir}/X11/bindings")
+XMBINDDIR_FALLBACK="$xmbinddir"
 AC_SUBST(XMBINDDIR_FALLBACK)
 
 RM="rm -f"
